launcher/src/main/java/org/apache/spark/launcher/package-info.java

```diff
@@ -17,10 +17,10 @@
 
 /**
  * Library for launching Spark applications.
- * <p/>
+ * <p>
```

Comment:
The most correct version of this and the previous edit above would be to wrap each paragraph in tags. Otherwise they run together in the javadoc as one paragraph, which is not so bad. Given that little of the Spark javadoc uses paragraph tags anyway, maybe just omit them rather than partly retain these line breaks? I'm happy to implement either one on merge or just leave it if nobody really cares.